### ISSUE TITLE
Incorrect value of data-custom-properties

### ISSUE THREAD

**Issue by AndriyBroda** [Choices-js/Choices#658]
When I set data-custom-properties for option in select
```
<select>
  <option value="Test1" data-custom-properties="testloremtext">Test1</option>
  <option value="Test2" data-custom-properties="test lorem text">Test2</option>
  <option value="Test3" data-custom-properties="{'description': 'item description'}">Test3</option>
</select>
```
Then here is the code in the output (the data-custom-properties value is obtained only up to the first space).

```
<select class="choices__input is-hidden" tabindex="-1" aria-hidden="true" data-choice="active">
  <option value="Test3" selected="" data-custom-properties="{'description':" 'item description' }>Test3</option>
</select>
```

**Comment by jshjohnson** [comment]
Hi, can you create a reduced test case illustrating the problem please

**Comment by cedbesafilm** [comment]
I think there are 2 problems

1. data-custom-properties are not return in event object
2. Even if i search the attribute in the element inside the event the value is trimmed

Example
```
<select id="nazione" data-function="test()">
    <option value="af" data-custom-properties="af<split>Afghanistan<split>Afghanistan">Afghanistan</option>
    <option value="al" data-custom-properties="al<split>Albania<split>Albania">Albania</option>
    <option value="aq" data-custom-properties="aq<split>Antartide<split>Antartide">Antartide</option>
</select>
```
and i select Afghanistan


Result
event.target.attributes.data-function.value = "test()"
event.target.selectedOptions[0.attributes.data-custom-properties.value = "af<split"

**Comment by tinovyatkin** [comment]
@jshjohnson that is fixed by #687

**Comment by cedbesafilm** [comment]
Only for knowledge

When the event it's fired, instead of check the event itself we use getValue() method of the instance the result is ok

![image](https://user-images.githubusercontent.com/55377336/67684919-5c44ce80-f994-11e9-9824-207ac2c6f158.png)
